tests/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtensionTest.php

```diff
@@ -516,6 +521,7 @@ private function getPartialContainerBuilderProphecy($test = false)
             'api_platform.listener.view.respond',
             'api_platform.listener.view.serialize',
             'api_platform.listener.view.validate',
+            'ApiPlatform\Core\Filter\QueryParameterValidateListener',
```

Comment:
Please change to `api_platform.listener.view.validate_query_parameters` for consistency (we don't use id as services name yet)

Reply:
Still, you could alias it if you think that it'd be used by developers.